### ISSUE TITLE
Where is Random Forest Algo used?

### ISSUE THREAD

**Issue by chakri7** [ShivamBhirud/Indian-Crime-Prediction-and-Visualization-Online-Platform#1]
Random Forest Algo is imported but never used , why and where was it intended to be used?

**Comment by ShivamBhirud** [comment]
Thanks for reaching out with a concern. I trained Linear regression and Random forest on my dataset to identify which ML model would be best suitable for my web app. I found out that random forest was somewhat inaccurate, it used to overfit where the data entries were consistent and used to underfit otherwise. However, that doesn't mean Linear regression was able to fit perfectly for my dataset every time. It was predicting well as compared to random forest. The reason for this was less volume of the dataset. In India, it seems that criminal records are not recorded in proper interval of time due to which the data that our govt. has is very less. So, I focused on predictions as much as possible but along with that, I tried to portray the patterns in criminal records from the year 1995 to 2017 approximately. Also, I am afraid I won't be able to provide the exact code and the analysis that was done between those two algorithms because it was done two years ago and while I pushed this project to the GitHub this is what I had with me on my system.
